diff --git a/include/urcu/arch/aarch64.h b/include/urcu/arch/aarch64.h
index 9e2f1ff..449c64b 100644
--- a/include/urcu/arch/aarch64.h
+++ b/include/urcu/arch/aarch64.h
@@ -42,14 +42,14 @@ extern "C" {
 #define __NR_membarrier		283
 #endif
 
-/*
- * https://gcc.gnu.org/bugzilla/show_bug.cgi?id=63293
- */
-#ifdef URCU_GCC_VERSION
-# if URCU_GCC_VERSION < 50100
-#  error Your gcc version performs unsafe access to deallocated stack
-# endif
-#endif
+// /*
+//  * https://gcc.gnu.org/bugzilla/show_bug.cgi?id=63293
+//  */
+// #ifdef URCU_GCC_VERSION
+// # if URCU_GCC_VERSION < 50100
+// #  error Your gcc version performs unsafe access to deallocated stack
+// # endif
+// #endif
 
 #ifdef __cplusplus
 }
